drivers/pcie/host/pcie.c

```diff
@@ -4,6 +4,13 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#if defined(CONFIG_PCIE_PRT)
+#if defined(CONFIG_X86_64)
+#define DT_DRV_COMPAT intel_pcie
+#else
+#error "ACPI DSDT not supported by current arch"
```

Comment:
so we could get rid of that message (which is a bit confusing anyway)

Reply:
updated